src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx

```diff
@@ -586,4 +614,123 @@ const Part: FunctionComponent<{
     </div>
   )
 }
+
+const Articles = ({articles, children}) => {
+  return (
+    <div className="bg-gray-50 dark:bg-gray-900 -mx-5 pt-24 pb-40 xl:px-0 px-5 text-white pb-12">
+      <div className="max-w-screen-lg mx-auto">
+        <div className="mb-4 uppercase font-medium tracking-wide text-sm md:text-left text-center text-blue-600">
+          Build Boyond this project
+        </div>
+        <h2 className="sm:text-2xl text-3xl md:text-left font-semibold text-center leading-tighter pb-4 dark:text-gray-200 text-gray-900">
+          Additional Learning Resources
+        </h2>
+
+        <div className="relative">
+          <div className="absolute inset-0">
+            <div className="h-2/3"></div>
+          </div>
+          <div className="relative max-w-7xl mx-auto">
+            <div className="mt-12 mx-auto grid gap-5 md:grid-cols-2 md:max-w-none">
+              {articles.map((article) => {
+                return (
+                  <div
+                    className="flex flex-col rounded-lg shadow-lg overflow-hidden mb-4"
+                    key={article.path}
+                  >
+                    <div className="flex-shrink-0">
+                      <img
+                        className="h-64 h-72 w-full object-cover"
+                        src={article.image}
+                        alt=""
+                      />
+                    </div>
+                    <div className="flex-1 dark:bg-gray-800 p-6 flex flex-col justify-between">
+                      <h2 className="uppercase font-semibold text-xs text-gray-700 dark:text-gray-200">
+                        {article.name}
+                      </h2>
+                      <div className="flex-1">
+                        <a href={article.path} className="block mt-2">
+                          <p className="text-xl font-semibold dark:text-white text-gray-900">
+                            {article.title}
+                          </p>
+                          <Markdown className="prose dark:prose-dark dark:prose-sm-dark prose-sm mt-4">
+                            {article.description}
+                          </Markdown>
+                        </a>
+                      </div>
+                    </div>
+                  </div>
+                )
+              })}
+            </div>
+          </div>
+        </div>
+        {children}
+      </div>
+    </div>
+  )
+}
+
+function FeaturedVideoCard({video}) {
```

Comment:
```suggestion
const  FeaturedVideoCard: React.FC<{video: any}> = ({video}) => {
```

Welcome to TypeScript